code/__DEFINES/maths.dm

```diff
@@ -6,6 +6,7 @@
 
 #define PI						3.1416
 #define INFINITY				1e31	//closer then enough
+#define system_type_infinity					1.#INF //only for isinf check
```

Comment:
Shouldnt we use full caps for this like other defines?

Reply:
You might be right at least for the sake of consistency.